### DISCUSSION TITLE
How do i increase player swimming speed?

### DISCUSSION THREAD

**Discussion by Tricked-dev** [Minestom/Minestom#1974] [Help]
Hi i tried changing the velocity but it only made things buggier (sinking the player) and didn't really work well since the x and z velocity are always 0.  
Changing the players speed attribute also doesn't affect swimming speed.  
Depth strider also doesn't work
So i am kinda out of idea's on how to implement this what are some possible solutions?  

Dolphins grace kinda works but only 1 level applies and add a bunch of gliding

**Comment by ex4dev** [comment]
Not really sure why, but when I gave depth strider boots to the player, changing the player's speed attribute started working in the water.

**Reply by Tricked-dev** [reply]
Can confirm this working, so odd lol but i'd rather find a way to do without but i guess this will do for now. Thanks!
